.eslintrc

```diff
@@ -1,7 +1,6 @@
 {
   "extends": "babel",
   "rules": {
-    "max-len": 0
```

Comment:
Shouldn't this configuration be in https://github.com/babel/eslint-config-babel ?